websub-ballerina/build.gradle

```diff
@@ -333,18 +344,27 @@ def getCheckedOutGitCommitHash() {
     'git rev-parse --verify --short HEAD'.execute().text.trim()
 }
 
+task createTestableJarZip(type: Zip) {
+    archiveName("${packageOrg}-${packageName}-${project.version}-testable-jars.zip")
+    destinationDir(file("${buildDir}/distributions"))
+    from ("${artifactTestableJar}") {
+        include '**/*-testable.jar'
+    }
+}
+
```

Comment:
We don't necessarily need to add this task. Since you've add it, let's remove the `getCheckedOutGitCommitHash()` task with this.

Reply:
* Removed `getCheckedOutGitCommitHash` from the build.